lapack/testlapack/general.go

```diff
@@ -1754,15 +1748,147 @@ func dlantr(norm lapack.MatrixNorm, uplo blas.Uplo, diag blas.Diag, m, n int, a
 					if math.IsNaN(sum) {
 						return sum
 					}
-					if sum > maxsum {
-						maxsum = sum
+					if sum > value {
+						value = sum
 					}
 				}
-				return maxsum
 			}
 		}
-	default:
-		// lapack.Frobenius
+	case lapack.Frobenius:
 		panic("not implemented")
 	}
```

Comment:
Add `default` back with a `panic("invalid norm")` (and below)?

Reply:
I've dropped the commit that modified `dlantr` because `dlansy` would need the same modification, it is not related to the purpose of the PR and doesn't add much value anyway. I've replaced it with a commit that adds the default cases as you suggested.